examples/express/server.ts

```diff
@@ -1,127 +1,114 @@
 import express from "express";
-import { GolemNetwork, Job, JobState } from "@golem-sdk/golem-js";
-import { getBlenderParams } from "./blenderUtils";
-import { fileURLToPath } from "url";
-const __dirname = fileURLToPath(new URL(".", import.meta.url));
+import { GolemNetwork, JobState } from "@golem-sdk/golem-js";
 
 const app = express();
-const port = 3001;
-app.use(express.json());
+const port = 3000;
 
-// serve our result images on localhost:3001/results/
-app.use("/results", express.static("public"));
+app.use(express.text());
 
-const network = new GolemNetwork({
-  // let's use one of ready-to-go images created by the Golem team
-  // if you want to use your own image take a look at https://handbook.golem.network/requestor-tutorials/vm-runtime
-  image: "golem/blender:latest",
-  // let's make sure the provider has enough resources to render our scene
-  demand: {
-    minMemGib: 1,
-    minStorageGib: 2,
-    minCpuThreads: 1,
-    minCpuCores: 1,
-  },
-  // upload the scene file before each job
-  beforeEachJob: async (workContext) => {
-    await workContext.uploadFile(`${__dirname}/cubes.blend`, "/golem/resource/scene.blend");
+const golemClient = new GolemNetwork({
+  yagna: {
+    apiKey: "try_golem",
   },
-  // if you want to see logs from the Golem node set this to true
-  enableLogging: false,
-  // by default Golem Network uses a simple in-memory storage for job statuses and results. In a real application you should use some persistent storage (e.g. a database)
-  // jobStorage: MyAwesomeJobStorage()
 });
 
-network.init();
-// network.init() happens asynchronously so let's create a middleware that will inform the user if the network is not ready yet
-app.use((_req, res, next) => {
-  if (!network.isInitialized()) {
-    res.status(503).send("Golem Network is not ready yet, please try again later");
-    return;
-  }
-  next();
-});
-
-app.post("/render-scene", async (req, res) => {
-  const frame = req.body.frame;
+await golemClient
+  .init()
+  .then(() => {
+    console.log("Connected to the Golem Network!");
+  })
+  .catch((error) => {
+    console.error("Failed to connect to the Golem Network:", error);
+    process.exit(1);
+  });
 
-  if (typeof frame !== "number") {
-    res.status(400).send("please specify which frame to render in the request body");
+app.post("/tts", async (req, res) => {
+  if (!req.body) {
+    res.status(400).send("Missing text parameter");
     return;
   }
+  const job = golemClient.createJob({
+    package: {
+      imageTag: "severyn/espeak:latest",
+    },
+  });
+
+  job.events.on("created", () => {
+    console.log("Job created");
+  });
+  job.events.on("started", () => {
+    console.log("Job started");
+  });
+  job.events.on("error", () => {
```

Comment:
maybe it would be natural to get an error in the parameter `job.events.on("error", (error) => {`